kalite/distributed/static/css/distributed/sidebar.css

```diff
@@ -24,9 +24,9 @@ Nav Panel
 
 .sidebar-panel {
     -webkit-transform: translateZ(0);
-    background: #5AA685;
+    background: #335c49;
```

Comment:
Please change the color back to the other one. It is part of our current color scheme.